chore(deps): bump github.com/hashicorp/terraform-plugin-sdk from 1.16.1 to 1.17.0 (#179)
spotinst/terraform-provider-spotinst: website/docs/r/ocean_aws.html.markdown

* `autoscale_is_enabled` - (Optional, Default: `true`) Enable the Ocean Kubernetes autoscaler.
* `autoscale_is_auto_config` - (Optional, Default: `true`) Automatically configure and optimize headroom resources.
* `autoscale_cooldown` - (Optional, Default: `null`) Cooldown period between scaling actions.

> * `auto_headroom_percentage` - (Optional) Set the auto headroom percentage, set a number between 0-200 to control the headroom % from the cluster. Relevant when isAutoConfig=true.

Reviewer: ```suggestion
* `auto_headroom_percentage` - (Optional) Set the auto headroom percentage (a number in the range [0, 200]) which controls the percentage of headroom from the cluster. Relevant only when `isAutoConfig` toggled on.
```
